alonet/detr/train.py

```diff
@@ -207,6 +197,8 @@ def build_criterion(
         loss_ce_weight=1,
         loss_boxes_weight=5,
         loss_giou_weight=2,
+        loss_dice_weight=0,
+        loss_focal_weight=0,
```

Comment:
Same, for me theses parameters should exists only in the detr_panoptic folder

Reply:
All these parameters were removing